server/opts.go

```diff
@@ -277,87 +277,89 @@ type AuthCallout struct {
-	TLSRateLimit          int64             `json:"-"`
+	ConfigFile             string        `json:"-"`
+	ServerName             string        `json:"server_name"`
+	Host                   string        `json:"addr"`
+	Port                   int           `json:"port"`
+	DontListen             bool          `json:"dont_listen"`
+	ClientAdvertise        string        `json:"-"`
+	Trace                  bool          `json:"-"`
+	Debug                  bool          `json:"-"`
+	TraceVerbose           bool          `json:"-"`
+	NoLog                  bool          `json:"-"`
+	NoSigs                 bool          `json:"-"`
+	NoSublistCache         bool          `json:"-"`
+	NoHeaderSupport        bool          `json:"-"`
+	DisableShortFirstPing  bool          `json:"-"`
+	Logtime                bool          `json:"-"`
+	LogtimeUTC             bool          `json:"-"`
+	MaxConn                int           `json:"max_connections"`
+	MaxSubs                int           `json:"max_subscriptions,omitempty"`
+	MaxSubTokens           uint8         `json:"-"`
+	Nkeys                  []*NkeyUser   `json:"-"`
+	Users                  []*User       `json:"-"`
+	Accounts               []*Account    `json:"-"`
+	NoAuthUser             string        `json:"-"`
+	SystemAccount          string        `json:"-"`
+	NoSystemAccount        bool          `json:"-"`
+	Username               string        `json:"-"`
+	Password               string        `json:"-"`
+	Authorization          string        `json:"-"`
+	AuthCallout            *AuthCallout  `json:"-"`
+	PingInterval           time.Duration `json:"ping_interval"`
+	MaxPingsOut            int           `json:"ping_max"`
+	HTTPHost               string        `json:"http_host"`
+	HTTPPort               int           `json:"http_port"`
+	HTTPBasePath           string        `json:"http_base_path"`
+	HTTPSPort              int           `json:"https_port"`
+	AuthTimeout            float64       `json:"auth_timeout"`
+	MaxControlLine         int32         `json:"max_control_line"`
+	MaxPayload             int32         `json:"max_payload"`
+	MaxPending             int64         `json:"max_pending"`
+	Cluster                ClusterOpts   `json:"cluster,omitempty"`
+	Gateway                GatewayOpts   `json:"gateway,omitempty"`
+	LeafNode               LeafNodeOpts  `json:"leaf,omitempty"`
+	JetStream              bool          `json:"jetstream"`
+	JetStreamMaxMemory     int64         `json:"-"`
+	JetStreamMaxStore      int64         `json:"-"`
+	JetStreamDomain        string        `json:"-"`
+	JetStreamExtHint       string        `json:"-"`
+	JetStreamKey           string        `json:"-"`
+	JetStreamOldKey        string        `json:"-"`
+	JetStreamCipher        StoreCipher   `json:"-"`
+	JetStreamUniqueTag     string
+	JetStreamLimits        JSLimitOpts
+	JetStreamTpm           JSTpmOpts
+	JetStreamMaxCatchup    int64
+	JetStreamMaxBufferMsgs int
+	JetStreamMaxBufferSize int64
```

Comment:
Should we have explicit json annotation tag?

Reply:
There's a comment further up on that struct saying the JSON tags were deprecated, so wasn't sure if it was needed.